common/aws/ecs/ecs.go

```diff
@@ -392,6 +392,12 @@ func (this *ECS) RunTask(cluster, taskDefinition string, count int64, startedBy
 		Count:          aws.Int64(count),
 		StartedBy:      startedBy,
 		Overrides:      taskOverride,
+		PlacementStrategy: []*ecs.PlacementStrategy{
```

Comment:
We will definitely need to separate that out given it's a huge feature request. The scheduler is only setup for binpack right now, and would scale very differently if someone wanted to use the `spread` strategy effectively.